data/remnant/remnant missions.txt

```diff
@@ -3948,8 +3948,20 @@ mission "Remnant: Cognizance 13"
 	
 
 event "remnant: postverta reveal"
-	system Postverta
-		remove hidden
+	unvisit Caeculus
+	unvisit Diespiter
+	unvisit Egeria
+	unvisit Levana
+	unvisit Prosa
+	unvisit Statina
+	link Postverta Diespiter
+	link Postverta Prosa
+	link Postverta Statina
+	system Caeculus
```

Comment:
I'd keep the system order where I had it. You've changed it to "five systems are being changed, here they are in alphabetical order," whereas previously I had "five systems are being changed in three different ways, here are those three ways [removing hidden, adding wormhole alpha, adding wormhole beta]."

Reply:
Ah, so that was the order. Sure, I will switch sorting orders.